src/assets/ar-busstops/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset='utf-8'>
+    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+    <title>GeoAR.js demo</title>
+    <script src='https://aframe.io/releases/1.0.0/aframe.min.js'></script>
+	<script src='https://unpkg.com/ar.js@^2.1/aframe/build/aframe-ar.min.js'></script>
+	<script src="https://unpkg.com/aframe-look-at-component@^0.8/dist/aframe-look-at-component.min.js"></script>
+    <script>
+        THREEx.ArToolkitContext.baseURL = 'https://unpkg.com/ar.js@^2.1/three.js/'
```

Comment:
I thought `three.js` is included as part of the `ar.js` as a dependency? aleast with npm install i see `three.js` popup in the package directory.

Reply:
It was in location based demos so I assumed it was required to make something work. App works fine without it so I have now removed it.